parm/nems.configure

```diff
@@ -2,14 +2,6 @@
 ####  NEMS Run-time Configuration File  #####
 #############################################
 
-# ESMF #
```

Comment:
@hu5970, do you have any reason for removing this part. This if-statement can keep the ufs-weather-model printing out a large number of PET* log files.

Reply:
Yes, this part somehow blocked OMP to use multiple threads. I do not know why but we have to live with a large number of PET* log files for now.

Reply:
@chan-hoo Are you okay with it despite this little annoyance?